src/azure-cli/azure/cli/command_modules/security/commands.py

```diff
@@ -261,5 +285,32 @@ def load_command_table(self, _):
         g.custom_command('list', 'list_security_sub_assessments')
         g.custom_command('show', 'get_security_sub_assessment')
 
+    with self.command_group('security iot-solution',
```

Comment:
could we add update command not integrate with create command for update operation?

Reply:
ASC asked to stay with the convention of create, delete, show, list 
and our REST have a createOrUpdate function meaning putting it on one category is problematic

Reply:
You can consider utilize generic_update_command to add `update` command with `PATCH-like` operation, which is consistent with CLI command guideline https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#standard-command-types.

Reply:
separated create and update as requested